contracts/minter-suite/Minters/MinterMerkle/MinterMerkleV1.sol

```diff
@@ -298,7 +309,24 @@ contract MinterMerkleV1 is ReentrancyGuard, IFilteredMinterMerkleV0 {
         payable
         returns (uint256 tokenId)
     {
-        tokenId = purchaseTo_K1L(msg.sender, _projectId, _proof);
+        tokenId = purchaseTo_K1L(msg.sender, _projectId, _proof, address(0));
+        return tokenId;
+    }
+
+    /**
+     * @notice Purchases a token from project `_projectId` as a delegate, (the
+     *         `msg.sender`) on behalf of an explicitly defined delegee.
+     * @param _projectId Project ID to mint a token on.
+     * @param _proof Merkle proof.
+     * @param _delegee Delegee being purchased on behalf of.
+     * @return tokenId Token ID of minted token
+     */
+    function purchase(
+        uint256 _projectId,
+        bytes32[] calldata _proof,
+        address _delegee
+    ) external payable returns (uint256 tokenId) {
+        tokenId = purchaseTo_K1L(msg.sender, _projectId, _proof, _delegee);
```

Comment:
I think minting to `msg.sender` is okay here, but we should make this behavior very clear on the frontend, because I can definitely see some users thinking the mint will be sent directly to their vault. We might even want to only accept vault to be specified on calls to `purchaseTo` (and assume delegee of zero here).
Honestly, this whole concept of "should a delegate claiming an airdrop/benefit of any kind be the receiver of said benefit" might be a good "best practice" question for the registry dev chat.

Reply:
The funds for minting are still coming from the delegate, right? If so then I agree it feels to me like our existing purchaseTo() and the delegate.cash purchasing are very similar user flows (difference being the delegation would automatically pull the registered vault address for the user instead of a copy/paste). As such I'm not sure if I see the use case where purchase() would need a `delegee` because implicitly this flow is always meant for sending $$ & receiving art to/from the same wallet.

^This has me wondering if we want to re-work the purchase modal along the lines of "purchase" and then a new screen or section that asks "are you sending this to another wallet?" --> if yes, then a copy/paste uses `purchaseTo()` or if getDelegatesForContract returns 1+ addresses, the delegate sees a list of vaults they can click & send to.

Reply:
Yeah this is a great point–wonder in this case if we may even want to just disallow `purchase` to be used when using this minter via the `delegate-vault` approach, instead _only allowing_ `purchaseTo`, just to simplify the set of possible interactions with this minter to be **only 3 options**:

1) minting as a directly-allowlisted minter (on the merkle list directly) using `purchase` (`msg.sender` receives NFT)
2) minting as a directly-allowlisted minter (on the merkle list directly) using `purchaseTo` (provided `_to` address receives NFT)
3) minting as the delegate for an allowlisted minter (vault is on merkle list, minter is not but is a delegate) using `purchaseTo` (provided `_to` address receives NFT)

>  Honestly, this whole concept of "should a delegate claiming an airdrop/benefit of any kind be the receiver of said benefit" might be a good "best practice" question for the registry dev chat.

This is a great point–going to post this convo there for any additional thoughts, and will also let this bop around in my brain a bit more over the weekend before I loop back and update this PR at all to account for this discussion and the others mentioned here.

Reply:
Updating to just support these 3 routes: `purchase`, `purchaseTo`, and `purchaseTo` as a delegate–I think this is the most explicit pattern and allows flexibility on the frontend for us to iterate on UX to make it as clear as possible.